sr_unix/ss_anal_shdw_file.c

```diff
@@ -82,15 +85,12 @@ void	ss_anal_shdw_file(char	*filename, int flen)
 	LSEEKREAD(shdw_fd, SNAPSHOT_HDR_SIZE, (sm_uc_ptr_t)(bitmap_buffer), bitmap_size, status);
 	if (0 != status)
 	{
+		free(bitmap_buffer);
```

Comment:
Since bitmap_buffer is malloced at line 83, can we remove the NULL initialization at line 55.

Reply:
Done.